README.md

```diff
@@ -1 +1,162 @@
- hello-microverse
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>Hello Microverse mini project</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [hello-microverse] <a name="about-project"></a>
+
+**[hello-microverse]** is a Microverse mini project to get done with the git and Github workflow.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Used html and css.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="">html</a></li>
+    <li><a href="">css</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="">Github</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Getting to know the double step saving process]** By using git we become familiar with the workflow of the DVCS.
+- **[Improve commit messages]** By using the workflow and keeping in mind persons that will need to review the code we write better commit messages.
+- **[Pull Request]** Finally learning how to use branches and pull requests.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://htmlpreview.github.io/github.com/flov3rh/hello-microverse/tree/main)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Project can be easily pulled and viewed in any web browser and code editor of choice.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+A web browser to access the live version or the cloned project.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:flov3rh/hello-microverse.git
+```
+
+### Usage
+
+To run the project, execute the following command simply open the live version or the index.html file.
+
+### Run tests
+
+To run tests, run the following command:
+
```

Comment:
- [ ] Kindly consider including the steps needed for users to run the test as you have included the text saying they should follow these steps but haven't included the steps.